nni/compression/pytorch/utils/shape_dependency.py

```diff
@@ -85,6 +85,10 @@ def reshape_break_channel_dependency(op_node):
     """
     in_shape = op_node.auxiliary['in_shape']
     out_shape = op_node.auxiliary['out_shape']
+    if not in_shape or not out_shape:
+        return True
```

Comment:
why in_shape/out_shape may be None? Should update the `graphutils.py` accordingly.